code/modules/mob/living/simple_animal/abnormality/waw/parasite_tree.dm

```diff
@@ -0,0 +1,382 @@
+	wander = 0
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 0.8)
+	ranged = TRUE
+	ranged_cooldown_time = 1 SECONDS
+	obj_damage = 0
+	del_on_death = TRUE
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	deathmessage = "shatters into numerous spongy splinters."
+	deathsound = 'sound/creatures/venus_trap_death.ogg'
+	attacked_sound = 'sound/creatures/venus_trap_hurt.ogg'
+	projectilesound = 'sound/machines/clockcult/steam_whoosh.ogg'
+	var/mob/living/simple_animal/hostile/abnormality/parasite_tree/connected_abno
+
+/mob/living/simple_animal/hostile/parasite_tree_sapling/Initialize()
+	..()
+	icon_living = "sapling[pick(1,2)]"
+	icon_state = icon_living
+	if(locate(/mob/living/simple_animal/hostile/abnormality/parasite_tree) in GLOB.abnormality_mob_list) //this is hopefully a short list.
+		connected_abno = locate(/mob/living/simple_animal/hostile/abnormality/parasite_tree) in GLOB.abnormality_mob_list
+		connected_abno.minions += src
+
+/mob/living/simple_animal/hostile/parasite_tree_sapling/death()
+	if(connected_abno)
+		connected_abno.minions -= src
+		connected_abno.endBreach()
+	for(var/atom/movable/AM in src)
+		AM.forceMove(get_turf(src))
+	..()
+
+/mob/living/simple_animal/hostile/parasite_tree_sapling/CanAttack(mob/living/carbon/human/the_target) //Your target has to be human and not have the tree curse.
+	if(isturf(the_target) || !the_target || the_target.type == /atom/movable/lighting_object) // bail out on invalids
+		return FALSE
+
+	if(ismob(the_target)) //Target is in godmode, ignore it.
+		var/mob/M = the_target
+		if(M.status_flags & GODMODE)
+			return FALSE
+
+	if(see_invisible < the_target.invisibility)//Target's invisible to us, forget it
+		return FALSE
+
+	if(ishuman(the_target) && the_target.stat != DEAD && !the_target.has_status_effect(THE_TREE_CURSE))
+		return TRUE
+	return FALSE
+
+/mob/living/simple_animal/hostile/parasite_tree_sapling/Move()
+	return FALSE
+
+/mob/living/simple_animal/hostile/parasite_tree_sapling/AttackingTarget(atom/attacked_target)
+	return OpenFire()
+
+/mob/living/simple_animal/hostile/parasite_tree_sapling/OpenFire()
+	if(ranged_cooldown > world.time)
+		return FALSE
+	ranged_cooldown = world.time + ranged_cooldown_time
+	playsound(get_turf(src), projectilesound, 10, 1)
+	var/smoke_test = locate(/obj/effect/particle_effect/smoke) in view(1, src)
+	if(!smoke_test)
+		var/datum/effect_system/smoke_spread/parasite_tree/S = new(get_turf(src))
+		S.set_up(5, get_turf(src))
+		S.start()
+		return TRUE
+	ranged_cooldown += 1 SECONDS
+
+//SMOKE EFFECT
+/obj/effect/particle_effect/smoke/parasite_tree
+	name = "thick noxious fumes"
+	color = "#AAFF00"
+	lifetime = 3
+	opaque = TRUE
+
+/obj/effect/particle_effect/smoke/parasite_tree/smoke_mob(mob/living/carbon/C)
+	if(!istype(C))
+		return FALSE
+	if(lifetime<1)
+		return FALSE
+	if(C.internal != null) //this part is edited so that rabbits with gas masks can be effected as well.
+		return FALSE
```

Comment:
Im not understanding the comment here, does this mean using a gas mask actually helps against the particles? If so I definitely approve of that.

Reply:
Oh yeah the default was that if you had a gas mask then you wouldnt be effected at all. I didnt know how prevelent gas masks were in the facility but ill test if simple masks also do this.